.moban.yaml

```diff
@@ -31,6 +31,7 @@ targets:
   - Gemfile: Gemfile.jj2
   - package.json: package.json.jj2
   - bear-requirements.txt: bear-requirements.txt.jj2
+  - Decription: Description.jj2
```

Comment:
please `git add Description` and commit it as part of your commit.

Reply:
I tried doing so but this error came up,
     fatal: pathspec 'Description' did not match any files
I was only able to execute the above command when i created a empty file of same name .